### ISSUE TITLE
Build failure with ghc 8.4

### ISSUE THREAD

**Issue by DanBurton** [ndmitchell/hoogle#245]
```
[ 6 of 28] Compiling General.Timing   ( src/General/Timing.hs, dist/build/General/Timing.o )

src/General/Timing.hs:72:46: error:
    Not in scope: data constructor ‘GCStats’
   |
72 |     s <- if not stats then return "" else do GCStats{..} <- liftIO getGCStats; return $ " (" ++ show peakMegabytesAllocated ++ "Mb)"
   |                                              ^^^^^^^
```

**Comment by ndmitchell** [comment]
Woohoo! That finally means enough of my dependencies are up to date! I'll fix this tomorrow.

**Comment by ndmitchell** [comment]
Fixed in the just released 5.0.17.2.
